pcsx2/Recording/InputRecordingControls.h

```diff
@@ -50,7 +52,7 @@ class InputRecordingControls
 	void Resume();
 	void SetFrameCountTracker(u32 newFrame);
 	// Sets frameAdvancing variable to false
-	// Used to restrict a frameAdvaceTravker value from transferring between recordings
+	// Used to restrict a frameAdvaceTracker value from transferring between recordings
```

Comment:
Thank you for fixing that before I did. lol

Reply:
Typo: frameAdvanceTracker , maybe capitalise on first letter too?

Reply:
The capitalization matches with the variable so no changes to make there, but yeah I didnt notice to other missing letter, I'll fix that lol